File: CS/Models/MyBinder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using DevExpress.Web.Mvc;
using System.ComponentModel;
using System.Web.Mvc;

public class MyEditorsBinder : DevExpressEditorsBinder {
    protected override object GetPropertyValue(ControllerContext controllerContext, ModelBindingContext bindingContext, PropertyDescriptor propertyDescriptor, IModelBinder propertyBinder) {
        if (propertyDescriptor.PropertyType == typeof(CommonMeals)) {
            TypeConverter tc = propertyDescriptor.Converter;
            return tc.ConvertFrom(EditorExtension.GetValue<String>(propertyDescriptor.Name));
        }

        return base.GetPropertyValue(controllerContext, bindingContext, propertyDescriptor, propertyBinder);
    }
}

File: VB/Models/MyBinder.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports DevExpress.Web.Mvc
Imports System.ComponentModel
Imports System.Web.Mvc

Public Class MyEditorsBinder
	Inherits DevExpressEditorsBinder
	Protected Overrides Function GetPropertyValue(ByVal controllerContext As ControllerContext, ByVal bindingContext As ModelBindingContext, ByVal propertyDescriptor As PropertyDescriptor, ByVal propertyBinder As IModelBinder) As Object
		If propertyDescriptor.PropertyType Is GetType(CommonMeals) Then
			Dim tc As TypeConverter = propertyDescriptor.Converter
			Return tc.ConvertFrom(EditorExtension.GetValue(Of String)(propertyDescriptor.Name))
		End If

		Return MyBase.GetPropertyValue(controllerContext, bindingContext, propertyDescriptor, propertyBinder)
	End Function
End Class

